edgegrid-signer-core/src/main/java/com/akamai/edgegrid/signer/EdgeGridV1Signer.java

```diff
@@ -16,6 +16,25 @@
 
 package com.akamai.edgegrid.signer;
 
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+import org.apache.commons.codec.binary.Base64;
```

Comment:
I was reading the docs on `DatatypeConverter` and I don't think I want to use it. For one thing, it's in the bit of the runtime library that's about XML processing, and we're not doing XML processing, so immediately it raises questions about what the heck this is doing in our not-XML-related code. The bigger concern I have is a note from the class's Javadoc:

> JAXB Providers are required to call the setDatatypeConverter api at some point before the first marshal or unmarshal operation (perhaps during the call to JAXBContext.newInstance). This step is necessary to configure the converter that should be used to perform the print and parse functionality.

I don't know what a JAXB provider is, if I have one, or who actually is responsible for making that method call to set this entire static class up to work. If that method isn't called then `#printBase64Binary()` is going to throw a `NullPointerException` (I read the class's code).